pkg/util/tracing/crdbspan.go

```diff
@@ -509,7 +510,11 @@ func (s *crdbSpan) recordFinishedChildren(childRecording tracingpb.Recording) {
 	// children being added to s.
 	for _, span := range childRecording {
 		for _, record := range span.StructuredRecords {
-			s.notifyEventListeners(record.Payload)
+			var d types.DynamicAny
+			if err := types.UnmarshalAny(record.Payload, &d); err != nil {
+				continue
+			}
```

Comment:
Perhaps it doesn't matter too much, but I kinda wonder if we can hold off on Unmarshaling this until we know there are event listeners in place for this span.

Reply:
I think we'd either have to change the `notifyEventListeners` signature to take a `*types.Any` or inside `notifyEventListeners` we will have to `MarshalAny` the passed in`item Structured` before we UnmarshalAny it into a DynamicAny. I think I'm going to leave it as is for now since this code is only called when importing remote spans into our span, not when every structured event is emitted.